### PR TITLE
Add "InfluxDB Cloud security" document

### PR DESCRIPTION
**Pull request by pierwill**
Closes https://github.com/influxdata/docs-v2/issues/2000.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud/reference/internals/security.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+  - [Advanced controls](#advanced-controls)
+- [Compliance auditing](#compliance-and-auditing)
+  to ensure continuous review and adoption of industry-recognized standards and best practices.
+
+## Tenant isolation
+
+In the InfluxDB Cloud multi-tenant platform, access controls ensure that only valid authenticated and authorized requests access your account data.
+Access control includes:
+
+- A unique organization ID is assigned to an account when the account is created.
```
Comment (sanderson) [inline content/influxdb/cloud/reference/internals/security.md:49]:
```suggestion
- A unique organization ID assigned to an account when the account is created.
```

File: content/influxdb/cloud/reference/internals/security.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+  All internal Cloud services require this organization ID to authenticate entities before accessing or operating on data.
+- All external requests must be authorized with a valid token or session.
+  This is enforced by every InfluxDB Cloud service, from individual API calls to UI interactions.
```
Comment (sanderson) [inline content/influxdb/cloud/reference/internals/security.md:52]:
```suggestion
  Every InfluxDB Cloud service, from individual API calls to user interface (UI) interactions, enforce this policy.
```

File: content/influxdb/cloud/reference/internals/security.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+
+## Data integrity
+
+A dedicated internal service ensures data integrity
+by periodically creating, recording, and writing test data into test buckets.
+The service then periodically executes queries to ensure the data hasn't been lost or corrupted.
+A separate instance of this service lives within each cluster.
```
Comment (sanderson) [inline content/influxdb/cloud/reference/internals/security.md:59]:
```suggestion
The service periodically executes queries to ensure the data hasn't been lost or corrupted.
A separate instance of this service lives within each InfluxDB Cloud cluster.
```

File: content/influxdb/cloud/reference/internals/security.md [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+Additionally, the service creates out-of-band backups in [line protocol](/influxdb/cloud/reference/syntax/line-protocol/),
+and ensures the backup data matches the data on disk.
+
+## Cloud infrastructure
+
+{{< img-hd src="/img/cloud-internals/cluster.png" alt="InfluxDB Cloud cluster architecture" />}}
```
Comment (sanderson) [inline content/influxdb/cloud/reference/internals/security.md:65]: Update image file path to use [image naming conventions](/influxdata/docs-v2/blob/master/CONTRIBUTING.md#image-naming-conventions):

```
/img/influxdb/cloud-internals-cluster.png
```

File: content/influxdb/cloud/reference/internals/security.md [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+
+Microsoft Azure data centers are compliant with many physical and information security standards.
+For detail about physical security within Microsoft Azure data centers, see [Microsoft's Compliance website](https://www.microsoft.com/en-us/trust-center/compliance/compliance-overview).
+
+### Data encryption
+
+InfluxDB Cloud enables TLS by default.
+TLS cannot be disabled.
+Data in transit to InfluxDB Cloud is encrypted using TLS from all clients, including Telegraf agents, browsers, and custom applications.
+Requests using TLS 1.2 or earlier are rejected.
```
Comment (sanderson) [inline content/influxdb/cloud/reference/internals/security.md:115]:
```suggestion
InfluxDB Cloud enforces TLS encryption for data in transit from all clients, including Telegraf agents, browsers, and custom applications.
Requests using TLS 1.2 or earlier are rejected.
```

File: content/influxdb/cloud/reference/internals/security.md [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+User managed encryption keys are not supported in InfluxDB Cloud at this time.
+
+## Application and service security
+
+### Internal access controls
+
+- Administrative privileges are restricted to named groups of authorized users.
+- Shared accounts are prohibited.
+- Multi-factor authentication (MFA) is required for all infrastructure (AWS, GCP, and Azure),
+  for all members of our GitHub organization,
```
Comment (sanderson) [inline content/influxdb/cloud/reference/internals/security.md:128]:
```suggestion
  for all members of the InfluxData GitHub organization,
```

File: content/influxdb/cloud/reference/internals/security.md [hunk 7]
```diff
@@ -0,0 +1,241 @@
...
+  and for other production systems with access to user information
+  (see ["InfluxData Subprocessors"](https://www.influxdata.com/legal/influxdata-subprocessors/)).
```
Comment (sanderson) [inline content/influxdb/cloud/reference/internals/security.md:130]:
```suggestion
  (see [InfluxData Subprocessors](https://www.influxdata.com/legal/influxdata-subprocessors/)).
```

File: content/influxdb/cloud/reference/internals/security.md [hunk 8]
```diff
@@ -0,0 +1,241 @@
...
+Developers use non-production environments solely for development, testing, and staging,
+and do not use user data for non-production use cases.
+Production access is limited to authorized personnel based on the principles of least privilege and separation of duties.
+
+### Monitoring, logging, and alerting
+
+- Continuous monitoring and metrics analysis of InfluxDB Cloud environments.
+- Services are carefully monitored to ensure performance and availability, and to detect anomalies.
+- Application logging, performance, and observability data are gathered and used for event analysis, capacity planning, alerting, and instrumentation.
+  Access to these logs and operator interfaces is controlled by group access permissions, and provided only to teams that require access to deliver InfluxDB Cloud services.
```
Comment (sanderson) [inline content/influxdb/cloud/reference/internals/security.md:168]:
```suggestion
InfluxData continuously monitors and analyzes metrics from InfluxDB Cloud environments.

- Services are carefully monitored to ensure performance and availability, and to detect anomalies.
- Application logging, performance, and observability data are gathered and used for event analysis, capacity planning, alerting, and instrumentation.
  Access to these logs and operator interfaces is controlled by group access permissions, and provided only to teams that require access to deliver InfluxDB Cloud services.
```

File: content/influxdb/cloud/reference/internals/security.md [hunk 9]
```diff
@@ -0,0 +1,241 @@
...
+### Incident response
+
+Report security incidents to <security@influxdata.com> or <support@influxdata.com>.
+We maintain procedures for incident alerting and response, performance or downtime events, security, and customer notification.
+We perform root cause analysis (RCA) as part of our incident response.
+
+## Configurable security controls
+### Access, authentication, and authorization
+
+{{< img-hd src="/img/cloud-internals/auth.png" alt="InfluxDB Cloud authentication and authorization" />}}
```
Comment (sanderson) [inline content/influxdb/cloud/reference/internals/security.md:205]: Update the image file path to follow image naming conventions:

```
/img/influxdb/cloud-internals-auth.png
```

File: content/influxdb/cloud/reference/internals/security.md [hunk 10]
```diff
@@ -0,0 +1,241 @@
...
+
+We use [Auth0](https://auth0.com/) for InfluxDB Cloud authentication.
+User accounts can be created directly on the InfluxDB Cloud system via Auth0.
+Users also have the option to use “social sign-on” with their existing Google or Microsoft accounts for authentication.
+API access within custom applications requires an authentication token.
+Tokens belong to an organization and are mapped to InfluxDB permissions within the organization as defined when the token is created.
+For more information on the types of tokens and ways to create them, please refer to our [documentation](/influxdb/cloud/security/tokens/).
```
Comment (sanderson) [inline content/influxdb/cloud/reference/internals/security.md:212]:
```suggestion
For more information on the types of tokens and ways to create them, see [Manage authentication tokens](/influxdb/cloud/security/tokens/).
```

File: content/influxdb/cloud/reference/internals/security.md [hunk 11]
```diff
@@ -0,0 +1,241 @@
...
+   If you would like to set this up, please contact us at <sales@influxdata.com>.
+- **Periodic API Token rotation**.
+   We recommend that users periodically connect directly from their applications
+   and automatically delete their current API tokens and create new tokens programmatically.
+   Please refer to our [API documentation](/influxdb/cloud/api/#tag/Authorizations)
+   for information on creating and deleting tokens programmatically.
+
+## Compliance and auditing
+
+InfluxDB Cloud is SOC2 Type II certified.
```
Comment (sanderson) [inline content/influxdb/cloud/reference/internals/security.md:240]:
```suggestion
InfluxDB Cloud is **SOC2 Type II** certified.
```

**Review by sanderson — CHANGES_REQUESTED** [review]

File: content/influxdb/cloud/reference/internals/security.md [hunk 12]
```diff
@@ -0,0 +1,240 @@
...
+Our security policy describes the secure development, deployment, and operation of InfluxDB Cloud.
+
+To protect data, InfluxDB Cloud includes the following:
+
+- Guaranteed [tenant isolation](#tenant-isolation) and [data integrity](#data-integrity).
+- Trusted cloud infrastructure
+  including [Amazon Web Services (AWS)](#amazon-web-services-aws),
+  [Microsoft Azure](#microsoft-azure),
+  and [Google Cloud Platform (GCP)](#google-cloud-platform-gcp),
+  building on [security controls of these cloud providers](#public-cloud-security),
```
Comment (kelseiv) [inline content/influxdb/cloud/reference/internals/security.md:24]: think we can drop this link?...no public cloud security section

File: content/influxdb/cloud/reference/internals/security.md [hunk 13]
```diff
@@ -0,0 +1,240 @@
...
+and ensures the backup data matches the data on disk.
+
+## Cloud infrastructure
+
+{{< img-hd src="/img/influxdb/cloud-internals-cluster.png" alt="InfluxDB Cloud cluster architecture" />}}
+
+InfluxDB Cloud is available on the following cloud providers:
+- Amazon Web Services (AWS)
+- Microsoft Azure
+- Google Cloud Platform (GCP)
```
Comment (kelseiv) [inline content/influxdb/cloud/reference/internals/security.md:70]: maybe nice to add links here?

File: content/influxdb/cloud/reference/internals/security.md [hunk 14]
```diff
@@ -0,0 +1,240 @@
...
+
+InfluxDB Cloud enforces TLS encryption for data in transit from all clients, including Telegraf agents, browsers, and custom applications.
+Requests using TLS 1.2 or earlier are rejected.
+
+By default, data at rest is encrypted using strong encrypted methods (AES-256) within AWS, GCP, and Microsoft Azure.
+
+User managed encryption keys are not supported in InfluxDB Cloud at this time.
+
+## Application and service security
+
```
Comment (kelseiv) [inline content/influxdb/cloud/reference/internals/security.md:120]: we may want to add an intro sentence (so we don't have double headers) --something like We observe the following application and service security measures:
- Internal access controls
- ....

File: content/influxdb/cloud/reference/internals/security.md [hunk 15]
```diff
@@ -0,0 +1,240 @@
...
+For more information, see [InfluxDB Cloud data durability](/influxdb/cloud/reference/internals/durability).
+
+### Incident response
+
+Report security incidents to <security@influxdata.com> or <support@influxdata.com>.
+We maintain procedures for incident alerting and response, performance or downtime events, security, and customer notification.
+We perform root cause analysis (RCA) as part of our incident response.
+
+## Configurable security controls
```
Comment (kelseiv) [inline content/influxdb/cloud/reference/internals/security.md:201]: same here, intro sentence

**Review by kelseiv — APPROVED** [review]

**Pull request merged**